tools/pot/openvino/tools/pot/engines/ie_engine.py

```diff
@@ -143,8 +143,12 @@ def predict(self, stats_layout=None, sampler=None, stat_aliases=None,
 
     @staticmethod
     def postprocess_output(outputs, _metadata):
-        """ Processes raw model output using the image metadata obtained during data loading """
-        return outputs
+        """ Processes model output data using the image metadata obtained during data loading
+        :param outputs: dictionary of output data per output name
+        :param _metadata: metadata obtained during data loading
+        :return: list of the output data sorted by output layers order.
```

Comment:
I think there is no, like "output layers order" for I suggest rephrasing the statement for the return value to 
`list of the output data in an order expected by the accuracy metric if any is used`